lib/services/android-project-service.ts

```diff
@@ -348,8 +368,12 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 	}
 
 	public async preparePluginNativeCode(pluginData: IPluginData, projectData: IProjectData): Promise<void> {
-		const pluginPlatformsFolderPath = this.getPluginPlatformsFolderPath(pluginData, AndroidProjectService.ANDROID_PLATFORM_NAME);
-		await this.processResourcesFromPlugin(pluginData, pluginPlatformsFolderPath, projectData);
+		if (!this.shouldUseNewRuntimeGradleRoutine(projectData)) {
+			const pluginPlatformsFolderPath = this.getPluginPlatformsFolderPath(pluginData, AndroidProjectService.ANDROID_PLATFORM_NAME);
+			await this.processResourcesFromPlugin(pluginData, pluginPlatformsFolderPath, projectData);
+		}
+
+		// Do nothing, the Android Gradle script will configure itself based on the input dependencies.json
```

Comment:
this comment looks strange - in case we enter the `if`, we'll still get here so it leaves me wondering what does `do nothing` mean

Reply:
The method wouldn't perform any work after the changes in the android build script, and would be completely empty after 4.0. I thought that maybe the comment would be relevant

Reply:
Ahh, I've missed we have this method in the interface and it is implemented for iOS. Ignore my comment :)